packages/main/test/specs/ColorPicker.spec.js

```diff
@@ -39,6 +39,32 @@ describe("Color Picker general interaction", () => {
 		assert.strictEqual(hexInput.getProperty("value"), "112233", "Shorthand syntax is supported");
 	});
 
+	it("Alpha value change", () => {
+		const colorPicker = browser.$("#cp1");
+		const alphaInput = colorPicker.shadow$("#alpha");
+
+		colorPicker.setProperty("color", "rgba(100, 100, 100, 1)");
+
+		alphaInput.click();
+		browser.keys(["Control", "A"]);
+		browser.keys("0");
+		browser.keys("Tab");
+
+		assert.strictEqual(colorPicker.getAttribute("color"), "rgba(100, 100, 100, 0)", "Alpha value propely changed");
+	});
+
+	it("Hue value change", () => {
```

Comment:
Please add a test with the same interaction for the alpha slider

Reply:
Done